app/lib/jsondiffpatch.coffee

```diff
@@ -0,0 +1,5 @@
+jsondiffpatch = require("bower_components/jsondiffpatch/public/build/jsondiffpatch-full.js")
+jsondiffpatch.formatters = require("bower_components/jsondiffpatch/public/build/jsondiffpatch-formatters.js")
+require("bower_components/jsondiffpatch/public/formatters-styles/html.css")
+
+module.exports = jsondiffpatch
```

Comment:
This logic was originally done by Bower overrides, and I *think* webpack ignores those? Not sure. Something to check.